internal/formatter/api_keys.go

```diff
@@ -0,0 +1,138 @@
+// Licensed to Yugabyte, Inc. under one or more contributor license
+// agreements. See the NOTICE file distributed with this work for
+// additional information regarding copyright ownership. Yugabyte
+// licenses this file to you under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package formatter
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+	ybmclient "github.com/yugabyte/yugabytedb-managed-go-client-internal"
+)
+
+const (
+	defaultApiKeyListing = "table {{.ApiKeyName}}\t{{.RoleName}}\t{{.ApiKeyStatus}}\t{{.Issuer}}\t{{.CreatedAt}}\t{{.LastUsed}}\t{{.ExpiryTime}}"
+	createdAtHeader      = "Date Created"
+	expiryTimeHeader     = "Expiration"
+	roleNameHeader       = "Role"
+	lastUsedHeader       = "Last Used"
+	createdApiKeyListing = "table {{.ApiKeyName}}\t{{.ExpiryTime}}\t{{.RoleName}}"
```

Comment:
Why do we need a separate one when we create?

Reply:
Not exactly needed. Just that the statusm issuer and last used are kind of redundant for a newly created API key
Should I just use the default?

Reply:
The 3 fields I've picked to show are to confirm the kind of generated API key to the user (info from the response)
When it'll actually expire, and what role it has (If not specified it'll be Admin so to let them know that etc)

Reply:
Done